tests/e2e/default/enketo/pregnancy.wdio-spec.js

```diff
@@ -31,8 +30,8 @@ describe('Pregnancy registration', () => {
   });
 
   it('Should submit a new pregnancy', async () => {
-    const edd = moment().add(30, 'days');
-    const nextANCVisit = moment().add(1, 'day');
+    const edd = moment().add(8, 'days');
+    const nextANCVisit = moment().add(2, 'day');
```

Comment:
I changed it because it was simpler for me to calculate the weeks of pregnancy 😅 if the delivery will happen in 8 days the woman will be 38 weeks pregnant. If leave it as it was, incrementing the number of days to 30 I will not know how many weeks she will be, because that will depend on the month and the amount of weeks that each month has.
I just tried to make it simpler 😅

Reply:
Great! thanks for the explanation!